snippets/ptp_PtpConfigGmWpc.yaml

```diff
@@ -0,0 +1,146 @@
+apiVersion: ptp.openshift.io/v1
```

Comment:
Need to add the snippet metadata:


Every text snippet should be placed in the snippets/ folder and should contain the following metadata at the top:

// Text snippet included in the following assemblies: (1)
//
// * list of assemblies where this text snippet is included
//
// Text snippet included in the following modules:    (2)
//
// * list of modules where this text snippet is included

:_mod-docs-content-type: SNIPPET     

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#text-snippet-file-metadata

Reply:
Not for YAML snippets though.